### ISSUE TITLE
Promise polyfill error on 2.4.4.

### ISSUE THREAD

**Issue by jinder** [Grsmto/simplebar#112]
Just upgraded to 2.4.4 on an Angular 4 project, which uses zone.js and am now seeing:

```
simplebar.js:12 Unhandled promise rejection Error: Zone.js has detected that ZoneAwarePromise `(window|global).Promise` has been overwritten.
Most likely cause is that a Promise polyfill has been loaded after Zone.js (Polyfilling Promise api is not necessary when zone.js is loaded. If you must load one, do so before loading zone.js.)
```

I think the library shouldn't be poly-filling Promise?

**Comment by Grsmto** [comment]
You're right it shouldn't but as it's using Promise and people are expecting a drop-in plugin that just works, I'm not sure how to handle that.
If Simplebar wants to support all the modern browsers, it still has to polyfill a bunch of things...
How libraries usually handle that kind of scenarios? I'll take a look!

**Comment by jinder** [comment]
I think if the user is downloading a file, they might expect it to be polyfilled.  However, for anyone installing as an NPM module, surely they can also add an es6-promise polyfill themselves if they need it?

**Comment by mnlfischer** [comment]
+1

**Comment by jinder** [comment]
@mnlfischer Awesome.  

I still think simplebar shouldn't be patching this in the first place.  Bloats the script size and it's something that can be easily polyfilled by the consumers of the library.

**Comment by kevinfaguiar** [comment]
I'm also trying to use this plugin in my Angular Cli (2+) project, and it is giving me this error: 
![pdc](https://user-images.githubusercontent.com/18222497/31238959-ed74d748-a9d1-11e7-8cb8-2ddb9cd7f2b4.PNG)
I think the errors are related.

**Comment by Grsmto** [comment]
Seems like the same issue as @jinder . Quite surprising for Angular to break on this tho but well.
I agree with everything said here and I'll definitely fix this asap.

**Comment by amarendrag** [comment]
I have same issue is there any update for the Angular issue

**Comment by tangopt** [comment]
Same here, just waiting for a fix

**Comment by Grsmto** [comment]
Thanks for your patience guys, this should be fixed in the latest 2.5.0.
